docs/user-guide/expressions/lists.md

```diff
@@ -1,47 +1,102 @@
 
-This context is ideal for computing things in row orientation.
+# Powerful *List* manipulation
 
-Polars expressions work on columns that have the guarantee that they consist of homogeneous data.
-Columns have this guarantee, rows in a `DataFrame` not so much.
-Luckily we have a data type that has the guarantee that the rows are homogeneous: `pl.List` data type.
+Let's say we had the following data from different weather stations across a state. When the weather station is unable to get a result, an error code is recorded instead of the actual temperature at that time.
 
-Let's say we have the following data:
+{{code_block('user-guide/expressions/lists','weather_df',['DataFrame'])}}
+```python exec="on" result="text" session="user-guide/lists"
+--8<-- "python/user-guide/expressions/lists.py:setup"
+--8<-- "python/user-guide/expressions/lists.py:weather_df"
+```
+
+## Creating a *List* column
 
-{{code_block('user-guide/expressions/lists','dataframe',['DataFrame'])}}
+For the `weather` `DataFrame` created above, it's very likely we need to run some analysis on the temperatures that are captured by each station. To make this happen, we need to first be able to get individual temperature measurements. This is done by:
 
+{{code_block('user-guide/expressions/lists','string_to_list',['str.split'])}}
 ```python exec="on" result="text" session="user-guide/lists"
---8<-- "python/user-guide/expressions/lists.py:setup"
---8<-- "python/user-guide/expressions/lists.py:dataframe"
+--8<-- "python/user-guide/expressions/lists.py:string_to_list"
+```
+
+One way we could go post this would be to convert each temperature measurement into its own row:
+
+{{code_block('user-guide/expressions/lists','explode_to_atomic',['list.explode'])}}
+```python exec="on" result="text" session="user-guide/lists"
+--8<-- "python/user-guide/expressions/lists.py:explode_to_atomic"
+```
+
+However, in Polars, we often do not need to do this to operate on the `List` elements.
+
+## Operating on *List* columns
+
+Polars provides several standard operations on `List` columns. If we want the first three measurements, we can do a `head(3)`. The last three can be obtained via a `tail(3)`, or alternately, via `slice` (negative indexing is supported). We can also identify the number of observations via `lengths`. Let's see them in action:
+
+{{code_block('user-guide/expressions/lists','list_ops',['list_ops'])}}
+```python exec="on" result="text" session="user-guide/lists"
+--8<-- "python/user-guide/expressions/lists.py:list_ops"
 ```
 
+You can check out all the operations that are available by looking through the [`List` API documentation](https://pola-rs.github.io/polars/py-polars/html/reference/expressions/list.html). If you find references to the `arr` API on Stackoverflow or other sources, just replace `arr` with `List`, this was the old accessor for the `List` datatype.
```

Comment:
Could you add the 'If you find references to the 'arr' sentence in an admonitions (https://squidfunk.github.io/mkdocs-material/reference/admonitions/#type:warning)?